tests/console/php_pcre.pm

```diff
@@ -31,7 +31,16 @@ sub run {
     assert_script_run "./test_pcrecpp";
     save_screenshot;
 
-    my $php = (is_leap('<15.0') || is_sle('<15')) ? 'php5' : 'php7';
+    my $php = '';
+    if (is_leap('<15.0') || is_sle('<15')) {
+        $php = 'php5';
+    }
+    elsif (is_sle('>=15-SP4')) {
```

Comment:
just wondering - is tumbleweed reporting true here? I thought it did not - which would imply TW uses php7 in this test, no? in main_common though, it would be php8

Reply:
is_sle returns 0 if distro != sle - so tumbleweed will indeed go the php7 case here, differing from what main_common does (php8 in TW case)

Reply:
Well it will not trigger, right. Anyway I'm not sure how to behave about TW right now, because php7 and the "basic" php8 tests are passing so, would it be advisable to change the schedule or just wait for it to be refined in the new [epic](https://progress.opensuse.org/issues/109569)?

Reply:
Confirmed on the provided test runs:

* http://10.161.229.196/tests/292#step/php_pcre/24 uses php7
* the rest of the test case all goes php8

Reply:
Ok, changed it to run php8 in TW and Leap 15.4 :smile: